lld/COFF/SymbolTable.cpp

```diff
@@ -669,8 +669,11 @@ bool checkLazyECPair(SymbolTable *symtab, StringRef name, InputFile *f) {
   if (std::optional<std::string> mangledName =
           getArm64ECMangledFunctionName(name))
     pairName = std::move(*mangledName);
+  else if (std::optional<std::string> demangledName =
+               getArm64ECDemangledFunctionName(name))
+    pairName = std::move(*demangledName);
   else
-    pairName = *getArm64ECDemangledFunctionName(name);
+    return true;
```

Comment:
What's the effect of returning true here?

Reply:
`checkLazyECPair` checks if there is a conflicting symbol with a different mangling in the symbol table before adding a new lazy symbol. Since invalid names cannot be mangled or demangled, there is no possibility of a conflicting symbol.

I updated the description.